tests/pz_pipeline_hsc.yaml

```diff
@@ -0,0 +1,41 @@
+description: |
+  Photo-z madness
+tasks:
+  pz_trainz:
+    class: lsst.meas.pz.estimate_pz_task.EstimatePZTask
```

Comment:
So, I see a number of cases of overriding setDefaults in config classes.   Are you saying that I should make additional Config / Pipetask class pairs for each algorithm?   So that each algorithm has four classes: a config / task pair to do the algorithm and config Pipetask pair to select the particular Task.   is that correct?